docs/changes.rst

```diff
@@ -23,6 +23,11 @@ Released: not yet
 
 **Incompatible changes:**
 
+* The `WBEMListener.start()` method may raise new exceptions
+  `pywbem.ListenerPortError`, `pywbem.ListenerPromptError` and
+  `pywbem.ListenerCertificateError`. For details, see the corresponding item
```

Comment:
we might comment that the old version raised IOError and OSError to cement the idea that this is an incompatibility.

Reply:
Well, they do replace the use of the earlier exceptions for the cases they are used for, but there is probably a chance that IOError on py27 and OSError still occur for some other cases that have not been replaced.
DONE - Added text: "The `OSError` and `IOError` exceptions raised in earlier versions may still be raised for other, less common cases."

Reply:
One more: I found that the new exceptions were not included in the docs yet. I added them to the indication llistener chapter, and improved their description to state when they were added and what the base class is, for consistency with our other exception classes.